src/coreclr/System.Private.CoreLib/src/ILLink/ILLink.Descriptors.Windows.xml

```diff
@@ -4,5 +4,7 @@
     <type fullname="System.Runtime.InteropServices.IDispatch" />
     <type fullname="System.Runtime.InteropServices.IMarshal" />
     <type fullname="Internal.Runtime.InteropServices.IClassFactory2" />
+    <!-- ComActivator is needed for native host operations -->
+    <type fullname="Internal.Runtime.InteropServices.ComActivator" />
```

Comment:
I thought we were going to put this behind the built-in COM feature switch?

Reply:
We cannot do that since that will trim away the methods needed by native COM hosting. See [above](https://github.com/dotnet/runtime/pull/54524#issue-674931772)

Reply:
Can we just root the methods that are necessary then, instead of the whole class?

Reply:
Just rooting the methods would also match what is done for other native hosting (`ComponentActivator` and `InMemoryAssemblyLoader`).

It should only need the three methods -  `GetClassFactoryForTypeInternal`, `RegisterClassForTypeInternal`, `UnregisterClassForTypeInternal`.